### ISSUE TITLE
Adding a CONTRIBUTING.md

### ISSUE THREAD

**Issue by Coding-Hashira** [Susmita-Dey/Moody#11]
Adding a Contributing guide can be helpful for beginners who are just starting with open source.

**Comment by Susmita-Dey** [comment]
@Coding-Hashira Go ahead

**Comment by Coding-Hashira** [comment]
> @Coding-Hashira Go ahead

I just gave the idea. I'm a beginner myself, so I don't think I'm the right person to add contributing guidelines to this project.

**Comment by Susmita-Dey** [comment]
@Coding-Hashira Okay then let's leave this issue for some other person. You can check my other projects **(except `Sukoon`)** to make contribution.

**Comment by mann-codes2014** [comment]
Hi, I am new to open source. But, I am interested in adding CONTRIBUTING.md. I can check other projects and make a pull request, then we can refine it, according to this project.

**Comment by Susmita-Dey** [comment]
@mann-codes2014 That's great. Go ahead.

**Comment by mann-codes2014** [comment]
Thanks, I will try to get it done as soon as possible.

**Comment by clairecharles** [comment]
Hey @Coding-Hashira i would love to work on this issue :)

**Comment by mann-codes2014** [comment]
Hey @clairecharles I am working on this.

**Comment by Coding-Hashira** [comment]
> @Coding-Hashira Okay then let's leave this issue for some other person. You can check my other projects **(except `Sukoon`)** to make contribution.

Can you please recommend me a good project to make meaningful contribution? I'm new to open source, that's why I'm having some difficulties with finding a project! 😅

**Comment by Susmita-Dey** [comment]
@Coding-Hashira You may participate in Hacktoberfest.

**Comment by Coding-Hashira** [comment]
> @Coding-Hashira You may participate in Hacktoberfest.

Yes, thinking to participate in it. Just thought to make some meaningful contributions before starting it! 😅

**Comment by Susmita-Dey** [comment]
@Coding-Hashira You'll become a pro by contributing in various projects during hacktoberfest

**Comment by Coding-Hashira** [comment]
@Susmita-Dey Looking Forward To It!!! 🤩🤩
